### PR TITLE
Feature/fix poa ui bugs cx 3229

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
This PR solves several UI issues on Safari iOS 7.
Text overflowing container (Safari - iPhone 5S - iOS 7)
screenshot 2019-02-25 at 18 50 43

Primary button outside of view width (Safari - iPhone 5S - iOS 7)
screenshot 2019-02-25 at 18 50 54

Icon too big, buttons hidden (iPhone SE)
image from ios 1

# Solution
The majority of the issues were due to the fact that Safari 6 on iOS 7 only supports `flexbox` with the `-webkit` prefix. 
We use autoprefixer plugin, therefore I added `iOS 7` to the list of supported browsers. As a bonus I removed `browsers` from the `autoprefixer()` options on `webpack.config.babel.js` as this is discouraged in the [autoprefixer documentation](https://github.com/postcss/autoprefixer#browsers) and I've add a `.browserslistrc` file.
The icons in the upload screen on small screen is now centered using flexbox.
See screenshots below.

iPhone SE
screen shot 2019-02-27 at 16 38 38

Safari - iPhone 5S - iOS 7
screen shot 2019-02-27 at 16 39 57

Safari - iPhone 5S - iOS 7
screen shot 2019-02-27 at 16 40 05
screen shot 2019-02-27 at 16 40 45

Bonus: fixes size of "Redo" button in confirm screen

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [x] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://591-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Confirm/index.js [hunk 1]
```diff
@@ -57,7 +58,7 @@ const Previews = localised(({capture, retakeAction, confirmAction, error, method
     translate(`confirm.${documentType}.message`)
 
   return (
-    <div className={classNames(style.previewsContainer, {
```
Comment (rfreitas) [inline src/components/Confirm/index.js]: what was the bug on the confirmation screen?

File: dist/onfido.min.js [hunk 2]
```diff
@@ -1,2 +1,2 @@
..._=a(g.length),k=0,w=n?p(t,_):c?p(t,0):void 0;_>k;k++)if((d||k in g)&&(m=g[k],v=b(m,k,y),e))if(n)w[k]=v;else if(v)switch(e){case 3:return!0;case 5:return m;case 6:return k;case 2:w.push(m)}else if(l)return!1;return f?-1:u||l?l:w}}},function(e,t,n){var r=n(42),o=n(16)("toStringTag"),i="Arguments"==r(function(){return arguments}());e.exports=function(e){var t,n,a;return void 0===e?"Undefined":null===e?"Null":"string"==typeof(n=function(e,t){try{return e[t]}catch(e){}}(t=Object(e),o))?n:i?r(t):"Object"==(a=r(t))&&"function"==typeof t.callee?"Arguments":a}},function(e,t,n){var r=n(15).document;e.exports=r&&r.documentElement},function(e,t,n){var r=n(42);e.exports=Array.isArray||function(e){return"Array"==r(e)}},function(e,t){e.exports=function(e,t){return{value:t,done:!!e}}},function(e,t){e.exports={"absolute-center":"onfido-sdk-ui-Video-absolute-center",challenge:"onfido-sdk-ui-Video-challenge",challengeTitle:"onfido-sdk-ui-Video-challengeTitle",introCopy:"onfido-sdk-ui-Video-introCopy",introBullets:"onfido-sdk-ui-Video-introBullets",introBullet:"onfido-sdk-ui-Video-introBullet",bolder:"onfido-sdk-ui-Video-bolder",introIcon:"onfido-sdk-ui-Video-introIcon",speak_out_loudIcon:"onfido-sdk-
```
Comment (rfreitas) [inline dist/onfido.min.js]: any reason for changing the dist file? it takes history space, so it's better if it's only changed for releases.

File: src/components/Confirm/index.js (hunk 1 above)
Comment (stefaniacardenas) [reply src/components/Confirm/index.js]: On iphone 5 you had to scroll down a bit to see the buttons. iPhone 5 screen is the smallest screen we support I believe so I've optimised in a way that the buttons will be fully visible on iPhone 5. On screens smaller than 320 x 568, the user might have to scroll down.

Before

screenshot 2019-03-04 at 12 25 16

After

screenshot 2019-03-04 at 12 31 13

File: dist/onfido.min.js (hunk 2 above)
Comment (stefaniacardenas) [reply dist/onfido.min.js]: No reason, I'll remove it.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by stefaniacardenas** [comment]
>what is the CSS size impact of this change?

It went from `139kb` to `141kb`

>Also does this include opera?

`last 2 versions` supports the last two versions of each browser in `browserslist` so yes, it includes Opera.
For reference 
https://github.com/postcss/autoprefixer#browsers
https://github.com/browserslist/browserslist

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by seewah** [comment]
buttons seem to be cropped in the mobile view too on chrome 
screen shot 2019-03-05 at 19 32 07

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by seewah** [comment]
sorry, another issue found. The redo button is too small on all the confirmation screens on mobile view (on all browsers) compare 
screen shot 2019-03-06 at 16 52 35 with 
screen shot 2019-03-06 at 16 52 48

**Comment by stefaniacardenas** [comment]
That issue with the error button size was outside of the scope of this ticket but as it was already in development. I've fixed it though, so this should be ready for review.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-591-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by seewah** [comment]
the enlarge image inspection screen is broken on iphone SE safari see 
screen shot 2019-03-07 at 09 37 57 I can reproduce with a real device on the office too

**Comment by seewah** [comment]
umm the above seems to be on development already. lets chat

**Pull request merged**